dashboard/app/models/levels/bubble_choice.rb

```diff
@@ -63,6 +63,13 @@ def sublevel_position(sublevel)
     i.present? ? i + 1 : nil
   end
 
+  # no need to call super or recurse here, because bubble choice levels and
+  # their children cannot have contained levels or project template levels.
```

Comment:
just to satisfy my own curiosity; what is it that prevents the child levels from having their own child levels, both here and in level groups?

Reply:
turns out I was mistaken here. I am updating this to take a recursive approach and will tag you again when it is ready for another look.